inc/dbmysql.class.php

```diff
@@ -1497,6 +1502,15 @@ public function beginTransaction() {
       return $this->dbh->begin_transaction();
    }
 
+   public function setSavepoint(string $name, $force = false) {
```

Comment:
If no transaction is started; and the `force` parameter is not used; then the method will not "work" without saying anything. I guess it should return an error in that case.